android/jni/app-android.cpp

```diff
@@ -438,7 +483,8 @@ bool System_GetPropertyBool(SystemProperty prop) {
 	case SYSPROP_HAS_IMAGE_BROWSER:
 		return true;
 	case SYSPROP_HAS_FILE_BROWSER:
-		return false;  // We kind of have but needs more work.
+		// return System_GetPropertyBool(SYSPROP_ANDROID_SCOPED_STORAGE);
```

Comment:
Oh, should we remove this line?  I think it makes sense even without scoped...

-[Unknown]

Reply:
Yeah... It doesn't hurt and seems to work well. So we might as well have it available on at least newer Android versions.